composer.json

```diff
@@ -33,19 +33,19 @@
         "psr-4": { "Omnipay\\Mollie\\Test\\": "tests/" }
     },
     "require": {
-        "omnipay/common": "^3"
+        "omnipay/common": "^3.0.2"
     },
     "require-dev": {
         "omnipay/tests": "^3",
         "squizlabs/php_codesniffer": "^3",
         "phpro/grumphp": "^0.14",
-        "phpmd/phpmd": "@stable",
-        "overtrue/phplint": "@stable",
-        "jakub-onderka/php-parallel-lint": "@stable"
+        "phpmd/phpmd": "^2",
+        "overtrue/phplint": "^1",
+        "jakub-onderka/php-parallel-lint": "^0"
```

Comment:
There is a 1.x release out: https://packagist.org/packages/jakub-onderka/php-parallel-lint